src/checker/Pulse.Syntax.Base.fsti

```diff
@@ -136,26 +135,28 @@ let ctag_of_comp_st (c:comp_st) : ctag =
   match c with
   | C_ST _ -> STT
   | C_STAtomic _ _ _ -> STT_Atomic
-  | C_STGhost _ -> STT_Ghost
+  | C_STGhost _ _ -> STT_Ghost
 
 noeq
 type effect_annot =
   | EffectAnnotSTT
-  | EffectAnnotGhost
+  | EffectAnnotGhost { opens:term }
   | EffectAnnotAtomic { opens:term }
+  | EffectAnnotAtomicOrGhost { opens:term }
```

Comment:
Effect annotation used to check with_invariants blocks, in case a post hint is not already present.